fmt/src/formatter.rs

```diff
@@ -1688,29 +1682,21 @@ impl<'a, W: Write> Visitor for Formatter<'a, W> {
         statements: &mut Vec<Statement>,
     ) -> Result<()> {
         if unchecked {
-            write!(self.buf(), "unchecked ")?;
+            write_chunk!(self, loc.start(), "unchecked ")?;
         }
 
         if statements.is_empty() {
             self.write_empty_brackets()?;
             return Ok(())
         }
 
-        let multiline = self.source[loc.start()..loc.end()].contains('\n');
+        writeln!(self.buf(), "{{")?;
```

Comment:
we should probably write any postfix comments before the opening bracket either with `write_postfix_comments` or implicitly with `write_chunk!`

Reply:
alternatively we could write postifx comments after the opening bracket, but we also have to make sure to cover the case of empty brackets and where to write the comments there if there are no statements, maybe using surrounded might help here, to handle that for you

Reply:
going w/ `write_chunk!`  here